microsoft-edge/devtools-guide-chromium/experimental-features/edge-command-palette.md

```diff
@@ -13,7 +13,7 @@ Use Command Palette to quickly access various Microsoft Edge browser commands an
 
 Using Command Palette, you can directly access productivity and developer features that normally require clicking through several menus or using a series of keyboard shortcuts.
 
-![A Microsoft Edge window showing the Command Palette input box in the center](./media/command-palette.png)
+![A Microsoft Edge window showing the Command Palette input box in the center](./edge-command-palette-images/command-palette.png)
```

Comment:
suggestion for possible consideration: 
global in images: remove all todo items, to remove distraction

Reply:
Good suggestion. In this case, because the website is rather narrow and to the side, and because we have red boxes around the important parts, I don't think those few TODO items distract too much. But I agree that, in general, we need to make sure the webpage is as undistracting as we can make it.
I'm not going to redo the images in this PR, but this is a great suggestion to keep in mind for future PRs. Thanks.